project.yaml

```diff
@@ -5,27 +5,27 @@ expectations:
 
 actions:
 
-  generate_base:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition_base
+  generate_main:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition_main
     outputs:
       highly_sensitive:
-        cohort: output/input_base.csv
+        cohort: output/input_main.csv
 
-  generate_study_population_total:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition
-    outputs:
-      highly_sensitive:
-        cohort: output/input.csv      
+  # generate_study_population_total:
+  #   run: cohortextractor:latest generate_cohort --study-definition study_definition
+  #   outputs:
+  #     highly_sensitive:
+  #       cohort: output/input.csv      
 
-  generate_study_population:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition --index-date-range "2021-01-01 to today by month" --skip-existing --output-dir=output/measures --output-format=csv.gz
-    outputs:
-      highly_sensitive:
-        cohort: output/measures/input_*.csv.gz
+  # generate_study_population:
+  #   run: cohortextractor:latest generate_cohort --study-definition study_definition --index-date-range "2021-01-01 to today by month" --skip-existing --output-dir=output/measures --output-format=csv.gz
+  #   outputs:
+  #     highly_sensitive:
+  #       cohort: output/measures/input_*.csv.gz
 
-  generate_measures:
-    run: cohortextractor:latest generate_measures --study-definition study_definition --skip-existing --output-dir=output/measures
-    needs: [generate_study_population]
-    outputs:
-      moderately_sensitive:
-        measure_csv: output/measures/measure_*.csv
+  # generate_measures:
+  #   run: cohortextractor:latest generate_measures --study-definition study_definition --skip-existing --output-dir=output/measures
+  #   needs: [generate_study_population]
+  #   outputs:
+  #     moderately_sensitive:
+  #       measure_csv: output/measures/measure_*.csv
```

Comment:
Don't be afraid to delete, rather than comment out bits of code. If you need the code again later, it can always be found in the commit history.